src/ProjectTemplates/Shared/Project.cs

```diff
@@ -156,11 +156,12 @@ internal async Task<ProcessResult> RunDotNetPublishAsync(IDictionary<string, str
 
         var result = new ProcessResult(execution);
 
+        // Comment out to see if it impacts test failures
```

Comment:
nit: Would be really surprising if this tiny bit of code caused the problems we're seeing😉

Reply:
Yeah I know. Just trying to see if any of the test additions are involved in the failures.